### DISCUSSION TITLE
Add functionality to subscribe to a thread to receive comment notifications

### DISCUSSION THREAD

**Discussion by darian-parrish** [deso-protocol/dips#240] [Medium CIPs]
**Purpose:** There is an interesting post and a user would like to receive notifications for all of the comments on that post.

**Details:** The subscribe function works both on posts and a comment(treats it as a post) and gives notifications for all children comments(but not grandchildren)

**Comment by matt-mcmarsh** [comment]
Love the idea
please make this happen core / nodes 🙏

**Comment by brootle** [comment]
I want to add that this should be optional, so kind of controlled by user in some settings and when user makes a comment he should be able to control the list of people who will be notified, so for example if he intends to reply to one person, he should be able to remove 6 other people from 'tag list' like this works on Twitter
